src/seedu/addressbook/ui/TextUi.java

```diff
@@ -169,4 +164,38 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * formatter class to format text for display
+     */
+    static class Formatter{
```

Comment:
Should have created a separate class file for Formatter.
Consider the case when you are working as a team or working on unit testing.